src/NuGet.Clients/NuGet.Console/OutputConsole/ChannelOutputConsole.cs

```diff
@@ -0,0 +1,149 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO.Pipelines;
+using System.Text;
+using System.Threading;
+using Microsoft;
+using Microsoft.ServiceHub.Framework;
+using Microsoft.VisualStudio;
+using Microsoft.VisualStudio.RpcContracts.OutputChannel;
+using Microsoft.VisualStudio.Shell.ServiceBroker;
+using Microsoft.VisualStudio.Threading;
+using IAsyncServiceProvider = Microsoft.VisualStudio.Shell.IAsyncServiceProvider;
+using Task = System.Threading.Tasks.Task;
+
+namespace NuGetConsole
+{
+    internal class ChannelOutputConsole : BaseOutputConsole, IDisposable
```

Comment:
Sealed?

Reply:
This will be extended eventually for https://github.com/NuGet/Home/issues/9270.